runtime/themes/kanagawa.toml

```
@@ -31,11 +30,11 @@
 "ui.bufferline.active" = { fg = "oldWhite", bg = "sumiInk0" }
 "ui.bufferline.background" = { bg = "sumiInk0" }
 
-"ui.popup" = { fg = "fujiWhite", bg = "sumiInk0" }
+"ui.popup" = { fg = "fujiWhite", bg = "sumInk0" }
```

```suggestion
"ui.popup" = { fg = "fujiWhite", bg = "sumiInk0" }
```

looks like a typo here